### PR TITLE
931 TC-WalkerColina-06

### PR DESCRIPTION
**Pull request by wcolina**
**Test Case:**
image

**Automated Test Case Result:**

image

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationCreate.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private WebElement descriptionTextInput;
+
+    @FindBy(id = "btnPpeClassificationSave")
+    private WebElement saveButton;
+
+    @FindBy(id = "btnPpeClassificationCancel")
+    private WebElement cancelButton;
+
+    public PPEClassificationCreate() {
+        CommonEvents.isPresent(saveButton);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationCreate.java]: use CommonEvents.isVisible(saveButton);

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private WebElement newPPEClassificationButton;
+
+    @FindBy(className = "container")
+    private WebElement ppeClassificationContainer;
+
+    @FindBy(xpath = "//table[@id='tblPpeClassification']/descendant::tr[last()]/td[count(//table[@id='tblPpeClassification']/descendant::th[text()='Name'])]")
+    private WebElement lastPPEClassificationName;
+
+    public PPEClassificationList() {
+        CommonEvents.isPresent(ppeClassificationContainer);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java]: usar CommonEvents.isVisible(ppeClassificationContainer);

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of cancel create ppe classification
+Feature: PPEClassificationCancelCreate
+
+  Scenario: La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'PPE Classification' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "fill"

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'Cancel' button into create 'PPE Classification' form page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+    Then Cancel creation 'PPE Classification' is not showed in PPE Classification list page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Corregir a minuscula "cancel"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 9]
```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of create ppe classification
+Feature: PPEClassificationCreate
+
+  Scenario: La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'PPE Classification' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "fill"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 12]
```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of create ppe classification
+Feature: PPEClassificationCreate
+
+  Scenario: La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE Classification' sub menu of 'PPE' menu
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
+    And Click 'Save' button into create 'PPE Classification' form page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationCreate.feature [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+    Then Created 'PPE Classification' is showed in PPE Classification list page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationCreate.feature]: Corregir a minuscula "create"

File: src/test/resources/features/ppe/PPEClassificationList.feature [hunk 15]
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of ppe classification list
+Feature: PPEClassificationList
+
+  Scenario: Clasificacion de equipo de proteccion personal - Todos los datos de clasificacion de equipos de proteccion personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationList.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEClassificationList.feature [hunk 16]
```diff
@@ -0,0 +1,9 @@
...
+    And Click 'PPE Classification' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEClassificationList.feature]: Corregir a minuscula "click"

**Review by airmiky — COMMENTED** [review]
Revisar observaciones por favor.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationCreate.java (hunk 1 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationCreate.java]: Done.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java (hunk 2 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 3 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 4 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 5 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 6 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 7 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCancelCreate.feature (hunk 8 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCancelCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 9 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 10 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 11 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 12 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 13 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationCreate.feature (hunk 14 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationCreate.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationList.feature (hunk 15 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationList.feature]: Done.

File: src/test/resources/features/ppe/PPEClassificationList.feature (hunk 16 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEClassificationList.feature]: Done.

**Comment by wcolina** [comment]
All comments fixed.

**Pull request merged**